test/pstl_offload/memory/usm_memory_replacement.cpp

```diff
@@ -160,5 +203,20 @@ int main() {
         EXPECT_TRUE(errno == ENOMEM, "Incorrect errno");
     }
 
+    // check that it's possible to release memory allocated via system allocation
+    {
+        allocs na;
+        perform_system_allocations(na);
+        check_memory_ownership(na, sycl::usm::alloc::unknown);
+        perform_deallocations_impl(na);
+    }
+    // check that system deallocation able to release memory allocated via overloaded allocation
```

Comment:
Actually, we don't test system deallocation function - it is our global replacement functions. May be it would be better to rewrite both comments to something like "Check the ability to deallocate USM memory from the translation unit built without PSTL offload mode"? What do you think?

Reply:
Good point, the terminology is not consistent. Are names below clear or something different it's better?
* native allocation
* global allocation
* locally-overloaded allocation

Reply:
I tried to rewrite the comments.